oca_dependencies.txt

```diff
@@ -0,0 +1,3 @@
+# List the OCA project dependencies, one per line
+# Add a repository url and branch if you need a forked version
+reporting-engine https://github.com/OCA/reporting-engine.git 10.0
```

Comment:
do we need this for this module?

Reply:
@elicoidal Yes, we will need this as 10.0 branch has some modules that depends on "report_py3o" module.
@victormartinelicocorp @seb-elico

Reply:
ok